python-sdk/nuimages/nuimages.py

```diff
@@ -8,17 +8,13 @@
 from collections import defaultdict
 from typing import Any, List, Dict, Optional, Tuple, Callable
 
-import cv2
 import matplotlib.pyplot as plt
 import numpy as np
 from PIL import Image, ImageDraw
 from pyquaternion import Quaternion
 
-from nuimages.utils.lidar import depth_map, distort_pointcloud, InvertedNormalize
 from nuimages.utils.utils import annotation_name, mask_decode, get_font
 from nuscenes.utils.color_map import get_colormap
-from nuscenes.utils.data_classes import LidarPointCloud
-from nuscenes.utils.geometry_utils import view_points, transform_matrix
 
 PYTHON_VERSION = sys.version_info[0]
 
```

Comment:
Should the message in `ValueError` be `nuImages dev-kit` instead of `nuScenes dev-kit`? (3 lines below this, 'cause I can't seem to write a comment on that exact line, haha)

Reply:
The pip package is still called `nuscenes-devkit`.